database/Constants/ScTrialDateLimits.cs

```diff
@@ -0,0 +1,8 @@
+namespace SCJ.Booking.Data.Constants
+{
+    public static class ScTrialDateLimits
```

Comment:
Looks great, but can we generalize the name of this file so it's an obvious place to put homeless constants?

For example, on WorkBC I had a file called `General.cs`
https://github.com/bcgov/workbc-jb/blob/master/src/WorkBC.Shared/Constants/General.cs

Maybe the variable could be called something like `ScMaxTrialDateSelections`?

Reply:
I'm renaming the file to `ScConstants` -- I thought about `ScGeneralConstants` but if it's already in the `Constants` namespace, hopefully it'll be clear enough. I'm not sure how that manifests in the editor!